examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<p>Wouldn't say "in progress" it's been around for quite a while an supports lots of case. Maybe just list some potential extensions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Dropped.